GaugeSupport.php

```diff
@@ -55,14 +55,14 @@ function menuLinks($p_event) {
 			array(
 				'title' => plugin_lang_get( 'menu_link' ),
 				'access_level' => '',
-				'url' => 'plugin.php?page=GaugeSupport/issue_ranking',
+				'url' => plugin_page( 'issue_ranking', true ),
 				'icon' => 'fa-line-chart'
 			),
 		);
 	}
 
 	function renderBugSnippet($p_event, $bugid) {
-		include 'plugins/GaugeSupport/pages/gauge_form.php';
+		include 'plugins/' . plugin_get_current() . '/pages/gauge_form.php';
```

Comment:
Your fix only goes half-way... What you should do to completely removed the hardcoded path is move `gauge_form.php` to a new _files_ directory, then
```suggestion
		include plugin_file_path( 'gauge_form.php', $this->basename );
```
This way the include will work as expected, even if the administrator has customized $g_plugin_path.